src/deploy-output.ts

```diff
@@ -18,6 +18,6 @@ export async function outputEnvironmentUrl(): Promise<void> {
     url = urls[0]
   }
   core.setOutput('deployed-url', url)
-  core.info(`Environment URL is ${url}`)
+  core.info(`Environment URL is ${url}.`)
```

Comment:
We can remove `.` as this line prints a URL and users may get confused considering `.` part of the URL.